ros/src/twist_controller/launch/dbw.launch

```diff
@@ -4,6 +4,7 @@
         <param name="vehicle_mass" value="1736.35" />
         <param name="fuel_capacity" value="13.5" />
         <param name="brake_deadband" value=".1" />
+        <param name="brake_factor" value="5.0" />
```

Comment:
Justin's graphs on our team channel show our brake values are close to the successful bag brake values. A factor of 5 may drastically change our Carla output. Should this factor be lower for Carla?

Reply:
I'll set that to 1.0 for Carla, so we can stick to the values found in the bag file. You're right, for the real car it's safer to go easy on the brakes.